<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fortepiano</class>
 <widget class="QMainWindow" name="Fortepiano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fortepiano</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>До</string>
    </property>
   </widget>
   <widget class="QLabel" name="DoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      [1; Q]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фортепиано</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      [2; W]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ре</string>
    </property>
   </widget>
   <widget class="QLabel" name="MiLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      [3; E]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FaButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MiButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ми</string>
    </property>
   </widget>
   <widget class="QLabel" name="FaLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>       [4; R]</string>
    </property>
   </widget>
   <widget class="QLabel" name="SolLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      [5; T]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SolButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Соль</string>
    </property>
   </widget>
   <widget class="QLabel" name="SiLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>       [7; U]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LjaButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SiButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Си</string>
    </property>
   </widget>
   <widget class="QLabel" name="LaLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      [6; Y]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="QStatusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
